vendor/sources/main.tf

```diff
@@ -51,9 +51,9 @@ locals {
         caching          = "ReadOnly,ReadOnly,ReadOnly,ReadOnly,ReadOnly,None"
         writeaccelerator = "false,false,false,false,false,false"
         luns             = "0,1,2#3#4#5"
-        names            = "data#log#shared#usrsap#backup"
+        names            = "datalog#shared#usrsap#backup"
```

Comment:
Should the data and logs names be separate here? as they have different paths: `/hana/data` and `/hana/log`

Reply:
No, this is on purpose. They create a unique `volume group` that later is splited in `70/30` space to provide for `/hana/data` and `/hana/log` respectively.